backend/server/tests/getAchievements.test.ts

```diff
@@ -0,0 +1,58 @@
+import { StateChallenge } from '../core/enums/stateChallenge.enum';
+import('../core/interfaces/challenge.interface');
+import { loadTasks } from './data/tasks';
+import { Task } from '../core/interfaces/task.interface';
+import { Challenge } from '../core/interfaces/challenge.interface';
+import { StateItem } from '../core/enums/stateItem.enum';
+import { Status } from '../core/interfaces/status.interface';
+import {getAchievements} from "../core/services/getAchievements.service";
```

Comment:
You need to use only one style in comment. You use space after `{` and  before `}` everywhere or you don't use it.
You use double quotes or single quotes everywhere.
Linter must say you about this. (Also linter may say about too long import paths and unused variables).